Examples/radial_distance_example.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+
+import parastell.parastell as ps
+import parastell.radial_distance_utils as rdu
+from parastell.utils import enforce_helical_symmetry, smooth_matrix
+
+
+# Define directory to export all output files to
+export_dir = ""
+# Define plasma equilibrium VMEC file
+vmec_file = "wout_vmec.nc"
+
+# Instantiate ParaStell build
+stellarator = ps.Stellarator(vmec_file)
+
+# Define build parameters for in-vessel components
+toroidal_angles = np.linspace(0, 90, num=97)
+poloidal_angles = np.linspace(0, 360, num=97)
+wall_s = 1.08
+# Define build parameters for magnet coils
+coils_file = "coils.example"
+width = 40.0
+thickness = 50.0
+toroidal_extent = 90.0
+
+# Measure separation between first wall and coils
+available_space = rdu.measure_fw_coils_separation(
+    vmec_file,
+    toroidal_angles,
+    poloidal_angles,
+    wall_s,
+    coils_file,
+    width,
+    thickness,
+    sample_mod=1,
+)
+# For matrices defined by angles that are regularly spaced, measurement can
+# result in matrix elements that are close to, but not exactly, helcially
+# symmetric
+available_space = enforce_helical_symmetry(available_space)
+# Smooth matrix
+available_space = smooth_matrix(available_space, 100, 1)
```

Comment:
I think it makes sense for the example to avoid magic numbers, maybe especially so - so that users can figure out what these are for